README.md

```diff
@@ -1,2 +1,237 @@
-# AwesomeModules
-This project stores the name and the author of a book. When the user logs can update and delete any book in the API based on its code. Easily add &amp; remove books from a list. Built with HTML, CSS &amp; JavaScript. Local Storage handling.
+<a name="readme-top"></a>
+
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
+
+Modify this file to match your project and remove sections that don't apply.
+
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+
+  <img src="./js/images/murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h2><b>Awesome Books</b></h2>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome Books] <a name="about-project"></a>
+
+
+**[Awesome Books]** "Awesome books" is a simple website that displays a list of books and allows you to add and remove books from that list.
+
+## 🛠 Built With <a name="built-with"></a>
+- HTML, CSS & Javascript
+- GitHub
+- Node
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+
+  </ul>
+</details>
```

Comment:
- [ ] I suggest that you remove this section since you're not using it in your README file, doing so will keep your documentation clean. 🤓 🔧